data/interactables.ron

```diff
@@ -3,12 +3,22 @@
 		Desk: (
 			position: (-27.0, -19.0),
 			collider_size: (3.0, 5.0),
-			range: 100.0,
+			range: 85.0,
 		),
 		Bed: (
 			position: (11.0, 5.0),
 			collider_size: (8.5, 4.5),
 			range: 60.0,
 		),
+		ClosedDoor: (
+			position: (-22.0, 6.5),
+			collider_size: (4.0, 1.0),
+			range: 19.0,
+		),
+		OpenDoor: (
+			position: (-22.0, 6.5),
+			collider_size: (4.0, 1.0),
+			range: 19.0,
+		)
```

Comment:
Can't we have a single door object with 2 states : opened & closed? We don't have many objects but maybe it's still better to aggregate them?

Reply:
I intended to do this at first, but I found it was easier to use two objects. Otherwise, I would have to look into disabling the collider and rigidbody or dealing with interaction groups. This would probably be the correct way to do it, but for the sake of speed, I chose to use two different objects.

Reply:
Ok I'm good with this :)